src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientTest.java

```diff
@@ -79,6 +86,14 @@ public class SnowflakeStreamingIngestClientTest {
   SnowflakeStreamingIngestChannelInternal<StubChunkData> channel3;
   SnowflakeStreamingIngestChannelInternal<StubChunkData> channel4;
 
+  // TODO: Add IcebergMode = True after Streaming to Iceberg is supported.
```

Comment:
lets make sure we return true her so that we're testing all throughout? The feature doesn't have to be released to customers / made public for testing to start?

Reply:
Added test parameter.